src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/DocumentBasedFixAllProvider.cs

```diff
@@ -72,97 +39,5 @@ internal abstract class DocumentBasedFixAllProvider : FixAllProvider
         /// <para><see langword="null"/>, if no changes were made to the document.</para>
         /// </returns>
         protected abstract Task<SyntaxNode?> FixAllInDocumentAsync(FixAllContext fixAllContext, Document document, ImmutableArray<Diagnostic> diagnostics);
-
-        private async Task<Document> GetDocumentFixesAsync(FixAllContext fixAllContext)
-        {
-            RoslynDebug.AssertNotNull(fixAllContext.Document);
-
-            var documentDiagnosticsToFix = await FixAllContextHelper.GetDocumentDiagnosticsToFixAsync(fixAllContext).ConfigureAwait(false);
-            if (!documentDiagnosticsToFix.TryGetValue(fixAllContext.Document, out var diagnostics))
-            {
-                return fixAllContext.Document;
-            }
-
-            var newRoot = await FixAllInDocumentAsync(fixAllContext, fixAllContext.Document, diagnostics).ConfigureAwait(false);
-            if (newRoot == null)
-            {
-                return fixAllContext.Document;
-            }
-
-            return fixAllContext.Document.WithSyntaxRoot(newRoot);
-        }
-
-        private async Task<Solution> GetSolutionFixesAsync(FixAllContext fixAllContext, ImmutableArray<Document> documents)
```

Comment:
this method was rewritten and is different in the new base class.  this original method tried to process all documents in the solution in parallel.  now we do things one project at a time.